modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -840,6 +884,27 @@ paths:
                     description:
                         default response
             tags: ["UserGroup"]
+    "/asset-libraries/pins":
+        get:
+            description:
+                Retrieves the pinned asset libraries.
+            operationId: getAssetLibrariesPinsPage
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                items:
+                                    $ref: "#/components/schemas/AssetLibrary"
+                                type: array
+                        application/xml:
+                            schema:
+                                items:
+                                    $ref: "#/components/schemas/AssetLibrary"
+                                type: array
+                    description:
+                        default response
+            tags: ["AssetLibrary"]
```

Comment:
For consistency with ratings (that work similarly), this should be called "pinned-by-me". Look for "rated-by-me" for examples.